packages/io/src/index.ts

```diff
@@ -7,3 +7,4 @@ export * from './controller'
 export * from './metadata'
 export * from './decorator'
 export * from './helper'
+export * from './nervos.chain-source'
```

Comment:
What naming style is this

Reply:
> What naming style is this

Inspired from NestJs, like `*.controller.ts`.

Reply:
Maybe it can be kept simple as `chain-source` for 2 reasons:

1. there isn't another chain as the source in the internal modules;
2. the name convention is rare in JavaScript ecosystem.

`chain-source.ts` can be extended to 

```
chain-source
├────index.ts
├────nervos.ts
└────others.ts
```

if necessary

Reply:
> Maybe it can be kept simple as `chain-source` for 2 reasons:
> 
> 1. there isn't another chain as the source in the internal modules;
> 2. the name convention is rare in JavaScript ecosystem.
> 
> `chain-source.ts` can be extended to
> 
> ```
> chain-source
> ├────index.ts
> ├────nervos.ts
> └────others.ts
> ```
> 
> if necessary

OK, I have changed like this early morning.